### PR TITLE
Fix #328: Add speaker details

### LINKED ISSUE #328: Add speakers Amit Bhattacharya, Vigneshwer Dhinakaran, and Emin Aksehirli
Amit Bhattacharya
His info is on https://eventyay.com/e/b548bcfc/#speakers
This is the photo:
image

Vigneshwer Dhinakaran
His info is on https://eventyay.com/e/b548bcfc/#speakers
There is one photo:
image
And another one on Google:
image
Which is better?

Emin Aksehirli
His info is on https://eventyay.com/e/b548bcfc/#speakers
This is the photo:
image

### PR DESCRIPTION
**Pull request by apooravc**
Added speakers: Amit Bhattacharya, Vigneshwer Dhinakaran, and Emin Aksehirli.
Fixes #328 
Preview: https://apooravc.github.io/2018.fossasia.org/

### CODE REVIEW

**Comment by apooravc** [comment]
Done. Please review.

**Comment by simsausaurabh** [comment]
Social links for Vigneshwer Dhinakaran redirects only to github remove the external link and leave the github link as it is. Rest is all fine. Do squash your commit after that.

File: index.html [hunk 1]
```diff
@@ -2623,6 +2623,59 @@ <h1>Speakers</h1>
...
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Vigneshwer Dhinakaran" src="img/VigneshwerDhinakaran.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://github.com/dvigneshwer"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://github.com/dvigneshwer"><i class="icon fa fa-github"></i></a>
```
Comment (bhaveshAn) [inline index.html]: GitHub profile referenced multiple times.

**Comment by vaibhavsingh97** [comment]
@apooravc Please fix conflicts

File: index.html [hunk 2]
```diff
@@ -2623,6 +2623,59 @@ <h1>Speakers</h1>
...
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                                <div class="image-holder">
+                                    <img class="background-image" alt="Vigneshwer Dhinakaran" src="img/VigneshwerDhinakaran.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://github.com/dvigneshwer"><i class="icon fa fa-external-link"></i></a>
```
Comment (aashutoshrathi) [inline index.html]: It's better to remove external link icon if we don't have one.

**Comment by hpdang** [comment]
thank you @apooravc

**Pull request merged**